tests/api_tests/test_score_modifiers_search.py

```diff
@@ -105,26 +133,9 @@ def test_valid_score_modifiers_format(self):
             }]
 
         for valid_score_modifiers in valid_score_modifiers_list:
-            modifiers_res = self.client.index(self.index_name_1).search(q=self.query, score_modifiers=valid_score_modifiers)
-
-    def test_bulk_search_error(self):
```

Comment:
Isn't this worth testing too? Why take this out?

Reply:
I don't really understand this test. It is either meant to
1. test bulk search with an invalid score modifier
2. Test that, since until now bulk search didn't support score modifiers, it would return an error (like an unsupported Error).
If its 1), then we already do that above, if its 2) its no longer needed and the test was implemented incorrectly because of the typo in `multiply_score_bys`